app/serializers/base_serializer.rb

```diff
@@ -0,0 +1,36 @@
+# Using https://github.com/fotinakis/jsonapi-serializers
+require 'jsonapi-serializers'
+
+class BaseSerializer < ActiveModel::Serializer
+  include JSONAPI::Serializer
+
+  def self_link
+    {
+      'api': "/api/v1#{super}",
+      'website': "#{super}"
+    }
+  end
+
+  def relationship_related_link(attribute_name)
+    "#{self_link[:api]}/#{format_name(attribute_name)}"
+  end
+
+  def relationship_self_link(attribute_name)
```

Comment:
Unused method argument - `attribute_name`. If it's necessary, use `_` or `_attribute_name` as an argument name to indicate that it won't be used. You can also write as `relationship_self_link(*)` if you want the method to accept any arguments but don't care about them.